open_mastr/xml_download/utils_write_to_database.py

```diff
@@ -302,44 +302,51 @@ def write_single_entries_until_not_unique_comes_up(
 
 
 def add_missing_column_to_table(
```

Comment:
I would suggest to rename it to add_missing_columns_to_table

Reply:
I agree and changed it.